Packs/Azure/Integrations/Azure/Azure.yml

```diff
@@ -1738,12 +1738,337 @@ script:
     - contextPath: Azure.CosmosDB.properties.writeLocations
       description: An array that contains the write locations enabled for the Cosmos DB account.
       type: List
+  - name: azure-billing-usage-list
+    description: Returns actual usage and cost details for a given time period, optionally filtered by service name.
+    arguments:
+      - name: subscription_id
+        description: Azure sub to run the command on - will be sent as scope parameter - /subscriptions/{subscriptionId}/.
```

Comment:
```suggestion
        description: The Azure subscription ID on which to run the command. It will be sent as a scope parameter - /subscriptions/{subscriptionId}/.
```